tmt/steps/discover/__init__.py

```diff
@@ -265,6 +279,7 @@ def extract_distgit_source(
             stdout, _ = self.run(
                 ["git", "config", "--get-regexp", '^remote\\..*.url'],
                 cwd=distgit_dir)
+            assert stdout is not None
```

Comment:
@janhavlin I would rather return a resonable error here? As I understand the code, this should be really handled as a reasonable error, it is an error condition to me.

Reply:
Changed it to raising a `GeneralError`. Is it okay now?